docker/Dockerfile

```diff
@@ -34,8 +34,8 @@ RUN --mount=type=bind,source=.git,target=${PUDL_REPO}/.git \
     mkdir ${CONTAINER_PUDL_IN} && mkdir ${CONTAINER_PUDL_OUT} && \
     # Run the PUDL setup script so we know where to read and write data
     ${CONDA_RUN} pudl_setup \
-        --pudl_in ${CONTAINER_PUDL_IN} \
-        --pudl_out ${CONTAINER_PUDL_OUT}
+    --pudl_in ${CONTAINER_PUDL_IN} \
+    --pudl_out ${CONTAINER_PUDL_OUT}
```

Comment:
Are these indents considered bad form or invalid? I was trying to make it clear when a new command in the long chain of `&&` commands was starting vs. new lines just being arguments.

Reply:
I'm not sure. My editor auto-corrected the indent. Happy to change it back.

Reply:
Hmm. Are you using VS Code? How do you have it set up to work with YAML?